dataikuapi/dss/projectdeployer.py

```diff
@@ -302,8 +357,28 @@ def get_raw(self):
         """
         return self.settings
 
+    def get_bundle_id(self):
+        """
+        Gets the bundle id currently used by this deployment.
+
+        :rtype: str
+        """
+        return self.settings["bundleId"]
+
+    def set_bundle_id(self, new_bundle_id):
+        """
+        Sets a new bundle id for this deployment. You need to call :meth:`~dataikuapi.dss.projectdeployer.DSSProjectDeployerDeployment.get_settings`
```

Comment:
I'm not so sure about how you suggest to make the change effective ^^.
You probably meant `DSSProjectDeployerDeploymentSettings.save()` ?